info.limpet.stackedcharts.core/src/info/limpet/stackedcharts/core/view/StackedChartsView.java

```diff
@@ -49,7 +50,9 @@
   // effects
   protected TransitionManager transitionManager = null;
   private Composite chartHolder;
-  private StackedchartsEditControl chartEditor;
+  private Composite editorHolder;
+  private ChartSet charts;
+  private AtomicBoolean initEditor = new AtomicBoolean(true);
```

Comment:
Aah, yes - you're performing two steps, but you want them to be treated as a single change.  I see.